presto-cli/src/main/java/io/prestosql/cli/AlignedTablePrinter.java

```diff
@@ -83,6 +83,64 @@ public void finish()
         writer.flush();
```

Comment:
can you please tell what improvements you get?

Reply:
Following is the average time taken by the query based on 3-4 runs on the data of ~160MB and 3000000 rows
CLI performance:
Before changes: 86 sec
After changes: 62 sec

JDBC performance:
Before changes: 106 sec
After changes: 90 sec

The average improvement in CLI is 28% and JDBC is 15%.